flocker/control/test/test_model.py

```diff
@@ -201,6 +243,53 @@ def test_applications(self):
                          sorted(list(node.applications) +
                                 list(another_node.applications)))
 
+    def test_update_node_new(self):
+        """
+        When donig ``update_node()``, if the given ``Node`` has hostname not
+        in existing ``Deployment`` then just add new ``Node`` to new
+        ``Deployment``.
+        """
+        node = Node(
+            hostname=u"node1.example.com",
+            applications=frozenset({Application(name=u'mysql-clusterhq',
```

Comment:
@exarkun is keen to remove mysql examples from the test suite. Maybe replace this one:
 * https://github.com/ClusterHQ/flocker/pull/1101/files#r23867389

...although I note that we forgot to update the mysql reference in that PR before merging.